spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,6 @@
+RSpec.describe ApplicationHelper, type: :helper do
+  describe '#format_currency' do
+    specify { expect(helper.format_currency(1_000_000)).to eq('£1,000,000') }
```

Comment:
Using the `it "explain behaviour" {}` syntax would improve documentation value of this test:

```rb
describe '#format_currency' do
  subject { helper.format_currency(1_000_000)

  context 'with a whole number' do
    let(:currency) { 1_000_000 }

    it 'removes the decimal digits' do
      expect(subject).to eq('£1,000,000')
    end
  end

  context 'with a decimal number' do
    let(:currency) { 1_000.50 }

    it 'persists the decimal digits' do
      expect(subject).to eq('£1,000.50')
    end
  end
end
```